instruments/mmoda_isgri/mmoda_isgri.install

```diff
@@ -45,12 +45,12 @@ function mmoda_isgri_install()
     'enabled' => TRUE,
     'name' => $instrument_info['name'],
     'title' => $instrument_info['title'],
-    'osa_version' => 'OSA11.1',
+    'osa_version' => 'OSA11.2',
```

Comment:
oh. and what are the possible options?

Reply:
ISGRI:
[https://github.com/oda-hub/mmoda-frontend-module/blob/ff1b8c0bb79cc03d67741fd3ee123f7405b82cab/instruments/mmoda_isgri/mmoda_isgri.inc#L42-L46](https://github.com/oda-hub/mmoda-frontend-module/blob/ff1b8c0bb79cc03d67741fd3ee123f7405b82cab/instruments/mmoda_isgri/mmoda_isgri.inc#L42-L46)

JEMX:
https://github.com/oda-hub/mmoda-frontend-module/blob/ff1b8c0bb79cc03d67741fd3ee123f7405b82cab/instruments/mmoda_jemx/mmoda_jemx.inc#L40-L43

Reply:
Some should be added?

Reply:
no, this is good.